sdk/lib/3rdparty/freetype/src/base/ftbitmap.c

```diff
@@ -500,11 +500,20 @@
 
   /* documentation is in ftbitmap.h */
 
+#ifdef __REACTOS__
+  FT_EXPORT_DEF( FT_Error )
+  FT_Bitmap_Convert_ReactOS_Hack( FT_Library        library,
+                                  const FT_Bitmap  *source,
+                                  FT_Bitmap        *target,
+                                  FT_Int            alignment,
+                                  FT_Bool           hack )
```

Comment:
I therefore wonder what kind of bitmap "format" is obtainend when hack is FALSE (i.e. what freetype was originally doing).

Reply:
Yep, it's what freetype was originally doing.